lighthouse-core/audits/accessibility/html-lang-valid.js

```diff
@@ -23,7 +23,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Specifying a valid [BCP 47 language](https://www.w3.org/International/questions/qa-choosing-language-tags#question) ' +
       'helps screen readers announce text properly. ' +
-      '[Learn more](https://dequeuniversity.com/rules/axe/4.4/html-lang-valid).',
+      '[Learn more about `lang` attributes](https://dequeuniversity.com/rules/axe/4.4/html-lang-valid).',
```

Comment:
doesn't match in construction the previous string (seems like this text should be the same for both)

(by previous string in this comment and others, I mean previous file in order of GH review)